compiler/circle-mpqsolver/src/core/Quantizer.h

```diff
@@ -53,6 +67,7 @@ class Quantizer
 private:
   std::string _input_dtype = "uint8";
   std::string _output_dtype = "uint8";
+  const QuantizerHook *_hook = nullptr;
```

Comment:
_hook is const but updated by set_hook?

Reply:
Yes. AFAIK it means that all hooks' functions should also be const.
In case i've used:
```
const QuantizerHook *const _hook = nullptr;
```
compiler would give an error in attempt to set `_hook` to another callback.
@jinevening
Should i change it to non-const?

Reply:
Ah, I misunderstood. You don't have to change it :)